modules/drone_odometry_local.py

```diff
@@ -37,24 +33,26 @@ def __init__(self,
         self.east = east
         self.down = down
 
-# pylint: enable=too-few-public-methods
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        return f"DronePositionLocal (NED): {self.north}, {self.east}, {self.down}"
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation